invenio_drafts_resources/resources/drafts/schema.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```

Comment:
Minor: I would not name it ``schema.py`` but ``links.py`` instead, as this is dealing specifically with links and the file can easily be confused with schema.py for the actual data schema and the two of them shouldn't really be mixed.

Reply:
split the difference here and named it: `schemas_links.py`